tests/js/spec/components/assistant/__snapshots__/helper.spec.jsx.snap

```diff
@@ -44,17 +45,87 @@ exports[`Helper renders guide drawer 1`] = `
         ],
       }
     }
-    onDismiss={[Function]}
-    onFinish={[Function]}
-    step={1}
-  />
+    onClick={[Function]}
+  >
+    <QuestionMarkIcon
```

Comment:
@adhiraj I'm surprised my change updated this jest snapshot. Does this seem correct to you?